css-brochure-form/styles.css

```diff
@@ -0,0 +1,55 @@
+* {
+  box-sizing: border-box;
+  font-family: BrandonGrotesque-Regular, sans-serif;
+}
+
+body {
+  margin-left: 27rem;
+  margin-right: 27rem;
```

Comment:
Instead of doing this, I suggest setting a `width` on the `form` element, then use `margin: auto` to center it 🤓